exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,33 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  // return num < 0;
+  if (num < 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
it can be more concise, can't it ? since the (num < 0) is already a boolean

Reply:
Do you mean something like return num <0;  I commented that out on line 9 because I thought we were supposed to use if else  statements.